js/views/pageNavVw.js

```diff
@@ -501,11 +501,13 @@ module.exports = baseVw.extend({
   },
 
   navBackClick: function(){
-    window.history.back();
+    history.action = 'back';
```

Comment:
I think this is fine as-is then. Good work, this is a smart way to fix this issue.